src/implementations/twig/components/spinner/spinner.story.js

```diff
@@ -0,0 +1,114 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import withCode from '@ecl/storybook-addon-code';
+
+import dataDefault from '@ecl/specs-component-spinner/demo/data';
+import spinner from './spinner.html.twig';
+import notes from './README.md';
+
+const dataNegative = { ...dataDefault, variant: 'negative' };
+
+const getArgs = (data, variant) => {
+  return {
+    text: data.text,
+    size: data.size || 'medium',
+    variant,
+    visible: true,
+    centered: true,
+    overlay: false,
+  };
+};
+
+const getArgTypes = () => {
+  return {
+    text: {
+      type: { name: 'string' },
+      description: 'Text below the loading indicator',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+    },
+    variant: {
```

Comment:
when selecting negative spinner on storybook (in the controls), we should apply a background (similar to what has been done for the links), otherwise it is not visible (in primary story)
Or we just don't show this control